src/pages/projectSettings/tabs/PluginsTab/getFormSchema.tsx

```diff
@@ -237,6 +255,11 @@ export const getFormSchema = (
           "ui:label": false,
         },
       },
+      ticketCreateIssueType: {
+        "ui:allowDeselect": false,
```

Comment:
The reason it isn't working is that `ui:allowDeselect` needs to be put on the `issueType` object, not the `ticketCreateIssueType` object. It will work after you make that change